Rimworld A13 update

Morgloz opened the pull request:
I updated the mod so it works in A13. I also tested it and seems fine.

PD:Since I had to modify some xml files I added the Def folder.

Morgloz committed:
Rimworld A13 update

jediminer543 commented:
I've tested with A13, and the duct system seems broken.

BeCracked commented:
Hey, I merged your changes manually and the textures are not quite working. Meaning objects show up in the UI and are placeable (though without any textures) and the typical red crosses in the UI.
I don't have much expertise in modding Rimworld so I wasn't able to fix it.
Quite a few mods (and CCL^^) are already installed. I will try again on a fresh install and comment again, if I can fix it.

Morgloz commented:
I posted in the forum how tu build the mod, this are only the folders to build the .dll plus the def folder, which I added because I changed some .xml files

Morgloz committed:
Ported to A14

Morgloz committed:
Ported to A14

Morgloz committed:
Ported to A14c

Morgloz committed:
Fixed medium heater bug

Morgloz committed:
Fixed textures

Morgloz committed:
Fixed buildings not working

Morgloz committed:
Improved description

Morgloz committed:
Fixed small cooler

Morgloz committed:
Added comp flickable

Morgloz committed:
Changed forum link

Morgloz committed:
Fixed industrial cooler side texture

Morgloz committed:
License updated

Morgloz committed:
Fixed small heater shadow

Morgloz committed:
Fixed texture loading

Morgloz committed:
Provisional rea

Morgloz committed:
Various fixes

Morgloz committed:
Removed CCL dependency

Morgloz committed:
Added interval configuration for wall checking

Morgloz committed:
Added wall checking

Morgloz committed:
A15 update

Morgloz committed:
Updated to A15c

Morgloz committed:
Changed research prerequisites

Morgloz committed:
Changed research prerequisites again

cainiaowu committed:
add chinese translation

Morgloz committed:
Merge pull request #2 from cainiaowu/test

add chinese translation

kphrx committed:
a16 update

kphrx committed:
Replaced WorkToMake with WorkToBuild

Morgloz committed:
Ported to A16

Morgloz committed:
merge with kPherox

Morgloz committed:
Merge branch 'kPherox-master'

Morgloz committed:
Fixed some errors

Morgloz committed:
Fixed Gizmo bug

Morgloz committed:
Added Multipurpose Cooler

Morgloz committed:
Fixes and added air flow units

Morgloz committed:
Fixed upper layer wrong display

Morgloz committed:
Ductwork reimplementation

Morgloz committed:
Intake fix

Morgloz committed:
Added flow to tooltip

duduluu committed:
Update Simplified Chinese and Add Traditional Chinese

ScottTRichmond committed:
Updated to a17 0.17.1557

Morgloz committed:
Merge pull request #8 from ScottTRichmond/master

Updated to A17

Morgloz committed:
Balance changes

Morgloz committed:
Merge pull request #5 from RimWorld-zh/master

Update Simplified Chinese and Add Traditional Chinese

Morgloz committed:
Merge branch 'master' of https://github.com/Morgloz/RimWorld-RedistHeat

Morgloz committed:
Nerfed Multipurpose Cooler

Morgloz committed:
Fixed ducts causing NaNC bug because of bad loading values

JonnyNova committed:
Fix wrong placement of debug message

JonnyNova committed:
Fix industrial cooler and exhaust ports incorrectly using the visible map

Morgloz committed:
Merge pull request #9 from JonnyNova/fix-exhaust-wrong-map

Fix industrial cooler losing exhaust ports when map is switched

JonnyNova committed:
Get layer for duct overlay from CompAir directly

Morgloz committed:
Merge pull request #10 from JonnyNova/aircomp-overlay-render

Get layer for duct overlay from CompAir directly

JonnyNova committed:
Add duct placeworker to industrial heater

(cherry picked from commit ea2db49)

JonnyNova committed:
Use below absolute zero for uninitialized temperature

JonnyNova committed:
Find ducts to wipe by class instead of defname, use map instead of visible map

JonnyNova committed:
Grab net temp from: air trader, intake room temp, then outside temp; use units instead of defname for pushers and pullers

JonnyNova committed:
Remove unused property

JonnyNova committed:
Correct flow calculation

Morgloz committed:
Merge pull request #11 from JonnyNova/serialize-airnet-temp

Get right air net temp on load and misc fixes

Morgloz committed:
Updated to A18 pending weird bug when placing Things on walls

Morgloz committed:
Fixed the weird bug and ductOutlet not being placeable

Morgloz committed:
Allowed side placing exhaust ports

Morgloz committed:
port to 1.0

Morgloz committed:
update target version
